shell/renderer/renderer_client_base.cc

```diff
@@ -156,6 +157,9 @@ void RendererClientBase::RenderThreadStarted() {
   extensions_renderer_client_.reset(new ElectronExtensionsRendererClient);
   extensions::ExtensionsRendererClient::Set(extensions_renderer_client_.get());
 
+  // Enables printing from Chrome PDF viewer.
+  pdf::PepperPDFHost::SetPrintClient(new ChromePDFPrintClient());
```

Comment:
NB: `SetPrintClient` stores the client in a thread local storage which doesn't do lifetime management, but in this case it is fine to leak as there is only one RenderThread per RenderClient in multi process mode and we don't support single process mode.